android-auto-app/src/main/AndroidManifest.xml

```diff
@@ -33,7 +28,7 @@
 
         <meta-data
             android:name="androidx.car.app.minCarApiLevel"
-            android:value="1"
+            android:value="3"
```

Comment:
can we still make it 1, so that out test app can run on older device versions? Can we make it backward compatible with earlier version?

Reply:
Yeah good call out. Best to keep the minCarApiLevel low. Also play close attention to `RequiresCarApi` annotation in case the sdk needs to condition on it.

But yes, the maps solution so far only needs api level 1 so reverting this change

Reply:
Oh here is why it was > 1. But there's no need for checking the required api so we're good here

```
    @RequiresCarApi(2)
    default void onScroll(float distanceX, float distanceY) {
    }
```